sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/SchemaRegistryAsyncClient.java

```diff
@@ -421,45 +398,34 @@ Mono<Response<SchemaProperties>> getSchemaPropertiesWithResponse(String groupNam
      *
      * @return The remapped error.
      */
-    private static Throwable remapError(ErrorException error) {
+    static HttpResponseException remapError(ErrorException error) {
         if (error.getResponse().getStatusCode() == 404) {
             final String message;
             if (error.getValue() != null && error.getValue().getError() != null) {
                 message = error.getValue().getError().getMessage();
             } else {
                 message = error.getMessage();
             }
-
             return new ResourceNotFoundException(message, error.getResponse(), error);
         }
 
         return error;
     }
 
     /**
-     * Converts an input stream into its string representation.
+     * Converts a Flux of Byte Buffer into its string representation.
      *
-     * @param inputStream Input stream.
+     * @param byteBufferFlux the Byte Buffer Flux input.
      *
      * @return A string representation.
      *
-     * @throws UncheckedIOException if an {@link IOException} is thrown when creating the readers.
      */
-    private static String convertToString(InputStream inputStream) {
+    static Mono<String> convertToString(Flux<ByteBuffer> byteBufferFlux) {
```

Comment:
I remember manually changing it to InputStream for a reason... I think it was trying to deserialize or reserialize the payload... but if this works in the integration tests then yay!